### PR TITLE
configuration_manual: http_storages: Add information about retry wait on errors

### PR DESCRIPTION
**Pull request by vmo69**
DOV-4331

### CODE REVIEW

File: source/configuration_manual/mail_location/obox/http_storages.rst [hunk 1]
```diff
@@ -82,7 +82,13 @@ The parameters common to all object storages include:
...
+| max_retries=<n>                       |Max number of HTTP request retries. Retries happen for 5xx errors as well as for 423(locked)                                   | 4            |
+|                                       |with :ref:`sproxyd <scality_sproxyd>` and 409(conflict) with :ref:`cdmi <scality_cdmi>`.                                       |              |
+|                                       |There is a wait between attempting next retry. The initial retry wait is 50ms, followed by doubling 100ms, 200ms, 400ms and    |              |
+|                                       |so forth.                                                                                                                      |              |
+|                                       |                                                                                                                               |              |
+|                                       |.. versionchanged:: 2.3.15 The new retry times are increased by multiplying with 10 e.g: 50ms -> 500ms -> 5s -> 10s. The       |              |
```
Comment (sirainen) [inline source/configuration_manual/mail_location/obox/http_storages.rst:90]: Would it make more sense to describe the new behavior in the main paragraph and the old behavior in this versionchanged line?

**Review by sirainen — CHANGES_REQUESTED** [review]

**Review by sirainen — APPROVED** [review]

File: source/configuration_manual/mail_location/obox/http_storages.rst [hunk 2]
```diff
@@ -82,7 +82,14 @@ The parameters common to all object storages include:
...
 +---------------------------------------+-------------------------------------------------------------------------------------------------------------------------------+--------------+
-| max_retries=<n>                       |Max number of HTTP request retries                                                                                             | 4            |
+| max_retries=<n>                       |Max number of HTTP request retries. Retries happen for 5xx errors as well as for 423(locked)                                   | 4            |
+|                                       |with :ref:`sproxyd <scality_sproxyd>` and 409(conflict) with :ref:`cdmi <scality_cdmi>`.                                       |              |
+|                                       |There is a wait between attempting next retry. The initial retry is done after 50ms. The following retries are done            |              |
+|                                       |after waiting ten times as long as the previous attempt, so 50ms -> 500 ms -> 5s ->10s. The maximum wait time before retry is  |              |
```
Comment (thuovila) [inline source/configuration_manual/mail_location/obox/http_storages.rst:88]: I think this refers to the absolute/cumulative total of 10 seconds? Could it be clarified with wording like "The maximum cumulative wait.."

Comment (sirainen) [reply source/configuration_manual/mail_location/obox/http_storages.rst:88]: No, in this case here it's a 10 second retry interval. There's absolute_timeout for the cumulative timeout. Wouldn't hurt to mention that of course.

Comment (vmo69) [reply source/configuration_manual/mail_location/obox/http_storages.rst]: The default(`max_retries=4`) cumulative wait is a always more then 15.55 seconds. If people configure more or less max_retries that can change.  I added "per attempt" to make it clearer.

If the request times out a warning is generated. If it times out depends on what is configured for `timeout`. I added a sentence about that as well. It's slightly longish now but i think that helps getting the whole picture.

Comment (vmo69) [reply source/configuration_manual/mail_location/obox/http_storages.rst]: Oh i did not see Timo's earlier comment. I will try update it with absolute_timeout.

Comment (vmo69) [reply source/configuration_manual/mail_location/obox/http_storages.rst]: Does it make sense now?

**Review by sirainen — APPROVED** [review]

**Review by thuovila — APPROVED** [review]

**Pull request merged**